src/dispatch/static/dispatch/src/case/TimelineTab.vue

```diff
@@ -2,23 +2,12 @@
   <v-container>
     <v-row justify="end">
       <v-switch v-model="showDetails" label="Show details" />
-      <v-btn
-        color="secondary"
-        class="ml-2 mr-2 mt-3"
-        @click="exportToCSV()"
-        :loading="exportLoading"
-      >
+      <v-btn color="secondary" class="ml-2 mr-2 mt-3" @click="exportToCSV()" :loading="exportLoading">
```

Comment:
Are you using Prettier and ESLint with your IDE? I think one-liners are harder to read and maintain.